src/snyk/snykOss/services/vulnerabilityCount/ossVulnerabilityCountService.ts

```diff
@@ -35,23 +40,21 @@ export class OssVulnerabilityCountService implements Disposable {
     private readonly workspace: IVSCodeWorkspace,
     private readonly window: IVSCodeWindow,
     private readonly languages: IVSCodeLanguages,
-    private readonly vulnerabilityCountProvider: ModuleVulnerabilityCountProvider,
+    private readonly vulnerabilityCountProvider: OssVulnerabilityCountProvider,
     private readonly ossService: OssService,
     private readonly logger: ILog,
     private readonly editorDecorator: EditorDecorator,
     private readonly analytics: IAnalytics,
     private readonly configuration: IConfiguration,
   ) {}
-
   activate(): boolean {
     this.disposables.push(
-      (this.diagnostics = this.languages.createDiagnosticCollection(DIAGNOSTICS_OSS_COLLECTION_NAME)),
```

Comment:
question: do we still create diagnostics for HTML somewhere?

Reply:
No, it looks like this is needed to show HTML diagnostics, I've reverted this change